### PR TITLE
Feature/add editorconfig

### PR DESCRIPTION
**Pull request by idormenco**
#232
add default editorconfig. 

default identation will be spaces.

### CODE REVIEW

**Comment by bvizureanu** [comment]
let's switch to the `develop`branch as default

**Comment by idormenco** [comment]
@bvizureanu done

File: src/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+# Core EditorConfig Options   #
+###############################
+# All files
+[*]
+indent_style = space
+# Code files
+[*.{cs,csx,vb,vbx}]
+indent_size = 4
+insert_final_newline = true
+charset = utf-8-bom
```
Comment (mihaipopescu) [inline src/.editorconfig]: I would recommend simply `utf-8`. I know a lot of tools (and windows) prefer `utf-8-bom` but the `BOM` is irrelevant for `utf-8`. see also this http://utf8everywhere.org/#faq.boms and this https://en.wikipedia.org/wiki/UTF-8#Byte_order_mark.

File: src/VotingIrregularities.sln [hunk 2]
```diff
@@ -41,7 +41,9 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "VoteMonitor.Api.Notificatio
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "VoteMonitor.Api.Statistics", "api\VoteMonitor.Api.Statistics\VoteMonitor.Api.Statistics.csproj", "{8BA1E3BC-0F4A-4120-965E-D2709887D05E}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "VoteMonitor.Api.DataExport", "api\VoteMonitor.Api.DataExport\VoteMonitor.Api.DataExport.csproj", "{4EDC6262-9BA5-46E7-BB90-E31F499A9F5A}"
```
Comment (mihaipopescu) [inline src/VotingIrregularities.sln]: I can't see how this change is related to this PR.

**Comment by adrianiftode** [comment]
Would you also consider having a separate PR that applies the editorconfig settings on each file under this repo? Thus will avoid future PRs having unrelated changes.

File: src/VotingIrregularities.sln (hunk 2 above)
Comment (idormenco) [reply src/VotingIrregularities.sln]: > I can't see how this change is related to this PR.
I wanted devs to see where do their VS gets formatting settings

**Comment by aniri** [comment]
@idormenco when you have some time, could you please update this? :pray:

**Comment by idormenco** [comment]
> @idormenco when you have some time, could you please update this? 🙏

@aniri i will close this pr and will add it again

**Pull request closed without merging**